walkthroughs/3-low-code-api-development/walkthrough.adoc

```diff
@@ -347,19 +347,17 @@ By default, Fuse Online does not expose an *Integration* to the public internet
 === Find the Greeting Service Internal Hostname
 
 . Login to the link:{openshift-console-url}[OpenShift Console, window="_blank"].
-. Select the project named with the following format: `fuse-<UNIQUE_ID>`.
-. Choose *Applications > Services* from the left menu.
-. Select *i-greeting-integration-{user-username}* from the *Services* list.
-. Note the *Hostname* listed on the *Service Details* screen. It will have a format similar to `i-greeting-integration-{user-username}.fuse-<UNIQUE_ID>.svc.cluster.local`
+. Select the project named with the following format: `redhat-rhmi-fuse`.
+. Choose *Network > Services* from the left menu.
+. Make sure there is a *i-greeting-integration-{user-username}* item in the *Services* list.
+. Take a note of the hostname: `i-greeting-integration-{user-username}.redhat-rhmi-fuse.svc.cluster.local`.
```

Comment:
QQ. Can the `svc.cluster.local` change? I feel as though I've also seen another variation of that now that I think of it.

Reply:
I think that value is for services inside a cluster. OS4 does no longer display the hostname in the service details, so I construct the URL directly.

Reply:
Gotcha, I think that's what I'm remembering - the 3 vs 4 difference 👍